dev-utils/src/commands/init_command.rs

```diff
@@ -67,6 +69,13 @@ impl InitCommand {
         self.remain_coin = (self.remain_coin - amount_coin).unwrap();
         self.distribution_addresses.push(address.to_string());
     }
+    fn read_chainid(&mut self) -> Result<(), Error> {
+        self.chainid = self.ask_string(
+            format!("new chain id( {} )=", self.chainid).as_str(),
+            self.chainid.as_str(),
+        );
+        Ok(())
+    }
     fn read_infomration_wallets(&mut self) -> Result<(), Error> {
```

Comment:
there's a typo in `information` -- but it's perhaps a bit redundant word (`read_wallets`, `read_genesis_time` etc.) is as good

Reply:
ok